src/request/components/request-detail-panel-logging.jsx

```diff
@@ -45,6 +46,30 @@ function getRowClass(message) {
     return rowClass;
 }
 
+function getDisplayText(level) {
+    return _.startCase(level);
+}
+
+function getIconName(level) {
```

Comment:
I agree with you, but we have been supplied with images we are meant to use for various reasons. I've got a copy of them which I can provide.